drivers/volume/portworx/portworx.go

```diff
@@ -1673,16 +1682,39 @@ func (d *portworx) UpgradeDriver(endpointURL string, endpointVersion string, ena
 		return fmt.Errorf("no endpoint supplied for upgrading driver")
 	}
 
-	if err := d.upgradePortworx(endpointURL, endpointVersion); err != nil {
-		return err
+	_, err := apiExtentions.GetCRD("storageclusters.core.libopenstorage.org", metav1.GetOptions{})
```

Comment:
Take a look at `sched-ops` we already implemented all the `StorarageCluster` operatons there, could use get from there https://github.com/portworx/sched-ops/blob/master/k8s/operator/storagecluster.go#L50